com/edocti/jintro/lab02/Ex7.java

```diff
@@ -0,0 +1,21 @@
+package com.edocti.jintro.lab02;
+
+import java.util.*;
+import java.text.*;
+
+public class Ex7 {
```

Comment:
Please use Ctr-Shift-f in eclipse to format the code correctly; or take care of formatting yourself (indentation, spaces, etc)